templates/todo/api/csharp-cosmos-sql-func/TodoList.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+using System.Text.Json.Serialization;
+
+namespace SimpleTodo.Api;
+
+public class TodoList
+{
+    public TodoList(string name)
+    {
+        Name = name;
+    }
+
+    [JsonPropertyName("id")]
+    public string? Id { get; set; }
+    [JsonPropertyName("name")]
```

Comment:
LIne break... last comment on that, just have a look at all files for places where blank lines are needed or not needed

Reply:
Resolved in latest commit.